snapshots/manager.go

```diff
@@ -44,7 +29,9 @@ type restoreDone struct {
 //    errors via io.Pipe.CloseWithError().
 type Manager struct {
 	store  *Store
+	opts   *types.SnapshotOptions
 	target types.Snapshotter
```

Comment:
out of curiosity, wdyt of the name target here?

Reply:
It makes sense to me because there are 2 stores within the `Manager`:
- `store  *Store` -used for persisting snapshots
- `target types.Snapshotter` - an interface but still a store.

The name `target` implies that the manager is taking snapshots from that `target` and storing them in the `store`

However, if you think that's confusing, I'm happy to change it so lmk

Reply:
Ahh that makes sense to me, can we add a comment on these fields explaining that?